### ISSUE TITLE
Allow re-running reference searches from command palette

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode-references-view#46]
Testing https://github.com/Microsoft/vscode/issues/64227.

Related to https://github.com/Microsoft/vscode-references-view/issues/45.

Right now I can only use the history after clearing the references search. Allow me to rerun a references search from the command palette no matter whether or not the references list shows results.

**Comment by jrieken** [comment]
Not yet sure how that would look. @kieferrm Are you thinking about a command that then shows quick pick? I am not sure what to put into those picks then? The word/line at which you have triggered those searches?

**Comment by kieferrm** [comment]
@jrieken yes, a command with a quick pick, the quick pick items should show the same information that is shown in the history of the References View.

**Comment by jrieken** [comment]
> the same information that is shown in the history of the References View.

That problem is that in there I cannot highlight the word from which you have started the last search. I can only the line without that
